functions/controllers/kaiAIController.js

```diff
@@ -264,6 +274,26 @@ app.post('/api/tool/', (req, res) => {
 
   bb.end(req.rawBody);
 });
+/**
+ * Save the tool session response to Firestore
+ * @param {object} sessionData - The data to be saved to Firestore
+ * @param {string} userId - The ID of the user
+ */
+const saveResponseToFirestore = async (sessionData) => {
+  try {
+    const toolSessionRef = await admin
+      .firestore()
+      .collection('toolSessions')
+      .add({
+        ...sessionData,
+        createdAt: Timestamp.fromMillis(Date.now()),
+      });
+
+    logger.log(`Tool session saved with ID: ${toolSessionRef.id}`);
```

Comment:
Let's set a flag called DEBUG in order to display this log when we are debugging, don't need this to log on every instance